src/platformIcon.tsx

```diff
@@ -97,7 +97,9 @@ export const PLATFORM_TO_ICON = {
   stride3d: "stride3d",
   swift: "swift",
   unity: "unity",
-  unreal: "unreal",
+  // This will be deprecated in favor of 'unrealengine'
+  ue4: "ue4",
+  "unrealengine": "unrealengine",
```

Comment:
```suggestion
  unrealengine: "unrealengine",
```

looks like we don't need `""` for the key